tests/shell/tests.py

```diff
@@ -94,3 +129,26 @@ def test_shell_with_bpython_not_installed(self, select):
     # returns EOF and so select always shows that sys.stdin is ready to read.
     # This causes problems because of the call to select.select() toward the
     # end of shell's handle() method.
+
+    def test_get_namespace(self):
+        from .models import Marker, Phone
```

Comment:
This inner import should not be required

Reply:
I tried to delete this imports but tests fail with errors like `NameError: 'Marker' is not defined`. Is there a way to import anything and make test pass?

Reply:
I meant you can do it at the top of the file, no need to do it inside the test function, called an "inner import". We limit inner imports to cases where it's required, such as fixing circular import errors.